root-packages/runc/fakes.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
```

Comment:
It would be better to rename this file to stubs.c.

Reply:
done